sssom/writers.py

```diff
@@ -77,13 +76,14 @@ def write_table(msdf: MappingSetDataFrame, filename: str, serialisation="tsv") -
 def write_rdf(
     msdf: MappingSetDataFrame,
     filename: str,
-    serialisation=SSSOM_DEFAULT_RDF_SERIALISATION,
+    serialisation: Optional[str] = None,
```

Comment:
rather than leave defaults in the function definiton, use None then assign the default inside. This means that any function calling this one doesn't have to repeat importing/using the default.